Shopify/cli
# Automatically remove trailing .js from @shopify/cli-kit imports

### WHY are these changes introduced?

I noticed that when we automatically import things from `@shopify/cli-kit` using the VSCode shortcut, it mistakenly adds a `.js` at the end of the module.

### WHAT is this pull request doing?

Adds a new eslint which can autofix the issue.

### How to test your changes?

- Try to import a function from a module you've never used before
- Command+S should autofix the import line

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
